### PR TITLE
Bug fixes02

### PR DESCRIPTION
**Pull request by ChoboHub**
Fix on extension.driver, update previousVersion needed false.
Review on Path URL and XPath edits for 2.6.7 compatibility #5 .

### CODE REVIEW

File: assets/extension_downloader.js [hunk 1]
```diff
@@ -117,7 +117,8 @@
 	var download = function (force) {
 		var data = {
 			q: input.val(),
-			force: force
+			force: force,
+	    	xsrf: Symphony.Utilities ? Symphony.Utilities.getXSRF() : ''
```
Comment (nitriques) [inline assets/extension_downloader.js]: Idendation is not right here. Can you please fix it ?

File: content/content.download.php [hunk 2]
```diff
@@ -133,16 +133,17 @@ private function download() {
 		private function searchExtension($query) {
 			
 			$xml = SymphonyExtensions::getExtensionAsXML($query);
+			$xml = $xml[0];
```
Comment (nitriques) [inline content/content.download.php]: Is this really needed ?

File: content/content.download.php [hunk 3]
```diff
@@ -133,16 +133,17 @@ private function download() {
...
 			
-			$this->extensionHandle = $xml->xpath('/response/extension/@id');
+			$this->extensionHandle = $xml->xpath('@id');
```
Comment (nitriques) [inline content/content.download.php]: I rather keep this... why is it needed ?

File: content/content.download.php [hunk 4]
```diff
@@ -133,16 +133,17 @@ private function download() {
...
 			
 			if (empty($this->extensionHandle)) {
 				throw new Exception(__("Could not find extension handle"));
 			} else {
 				$this->extensionHandle = (string)$this->extensionHandle[0];
 			}
 			
-			$this->downloadUrl = $xml->xpath("/response/extension/link[@rel='github:zip']/@href");
+			$this->downloadUrl = $xml->xpath("link[@rel='github:zip']/@href");
```
Comment (nitriques) [inline content/content.download.php]: Same here!

**Comment by ChoboHub** [comment]
All reviewed

File: content/content.download.php (hunk 2 above)
Comment (nitriques) [inline content/content.download.php]: Please use this instead

```php
$xml = current($xml);
```

**Comment by nitriques** [comment]
Cool! The only thing missing is editing the meta.xml file to create a new release with today's date and a description of what got fixed!

File: extension.meta.xml [hunk 5]
```diff
@@ -23,6 +23,9 @@
 		<!-- None -->
 	</dependencies>
 	<releases>
+		<release version="1.2" date="2016-07-21" min="2.3" max="2.x.x">
+			- Updated fixed issues #5 and #6, compatibility with php 7
```
Comment (nitriques) [inline extension.meta.xml]: Version should be 1.1.2 ;)

Comment (nitriques) [inline extension.meta.xml]: We try to follow http://semver.org/

File: extension.meta.xml [hunk 6]
```diff
@@ -26,9 +26,6 @@
 		<release version="1.1.2" date="2016-07-21" min="2.3" max="2.x.x">
 			- Updated fixed issues #5 and #6, compatibility with php 7 
 		</release>
-		<release version="1.1.1" date="2016-03-07" min="2.3" max="2.x.x">
-			- Updated compatibility info
-		</release>
```
Comment (nitriques) [inline extension.meta.xml]: This is good! It should be kept.

File: extension.meta.xml [hunk 7]
```diff
@@ -23,8 +23,11 @@
 		<!-- None -->
 	</dependencies>
 	<releases>
+		<release version="1.1.2" date="2016-07-21" min="2.3" max="2.x.x">
+			- Updated fixed issues #5 and #6, compatibility with php 7 
+		</release>
 		<release version="1.1.1" date="2016-03-07" min="2.3" max="2.x.x">
-			- Updated compatibility info
+			 Updated compatibility info
```
Comment (nitriques) [inline extension.meta.xml]: Haha, still missing a - !

It's important since the content of the xml tag will be parsed by markdown!

**Comment by nitriques** [comment]
Merged as edc5667

**Pull request merged**